samples/extensions/shader_debugprintf/shader_debugprintf.cpp

```diff
@@ -433,23 +433,30 @@ const std::vector<const char *> ShaderDebugPrintf::get_validation_layers()
 // This sample overrides the instance creation part of the framework to chain in additional structures
 std::unique_ptr<vkb::Instance> ShaderDebugPrintf::create_instance(bool headless)
 {
-	uint32_t instanceApiVersion;
-	VK_CHECK(vkEnumerateInstanceVersion(&instanceApiVersion));
+	uint32_t layer_property_count;
```

Comment:
Can you add a bit of an explanation in here as to why this is needed? That would make it easier to follow the code.

Reply:
Done